### PR TITLE
Entry point to loyalty rewards ("explore rewards" link on Overview and Settings)

### PR DESCRIPTION
**Pull request by vyorkin**

### CODE REVIEW

File: src/ui/pages/Overview/Overview.tsx [hunk 1]
```diff
@@ -227,6 +229,28 @@ function CurrentAccountControls() {
...
+function RewardsLinkIcon() {
+  return (
+    <Button
+      kind="ghost"
+      as={UnstyledAnchor}
+      href="https://app.zerion.io/rewards"
+      target="_blank"
+      size={36}
+      title="Rewards"
+      style={{ paddingInline: 8 }}
```
Comment (zerts) [inline src/ui/pages/Overview/Overview.tsx:241]: `rel="noopener noreferrer"`

File: src/ui/pages/Settings/Settings.tsx [hunk 2]
```diff
@@ -99,6 +100,34 @@ function SettingsMain() {
...
+        <Frame>
+          <VStack gap={0}>
+            <FrameListItemAnchor
+              href="http://app.zerion.io/rewards"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <AngleRightRow kind="link">
+                <HStack gap={8} alignItems="center">
+                  <RewardsIcon />
```
Comment (zerts) [inline src/ui/pages/Settings/Settings.tsx:112]: Looks like something aligned differently
Zerts Screenshot 2024-11-07 at 15 06@2x

**Review by zerts — APPROVED** [review]

File: src/ui/pages/Overview/Overview.tsx (hunk 1 above)
Comment (vyorkin) [reply src/ui/pages/Overview/Overview.tsx:241]: oh, thanks!

File: src/ui/pages/Settings/Settings.tsx (hunk 2 above)
Comment (vyorkin) [reply src/ui/pages/Settings/Settings.tsx]: thanks! Fixed ✅

**Pull request merged**